recipes/simdjson/0.4.7/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8)
+project(cmake_wrapper)
+
+include("conanbuildinfo.cmake")
+conan_basic_setup()
+if(WIN32 AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+endif()
```

Comment:
```suggestion

```

I don't think you need to force it as simdjson already exposed it: https://github.com/simdjson/simdjson/blob/v0.4.7/src/CMakeLists.txt#L41